arch-chroot.in

```diff
@@ -2,27 +2,30 @@
 
 shopt -s extglob
 
+# m4_include() is recognized as a function definition
+# shellcheck source=common disable=SC1073,SC1065,SC1064,SC1072
 m4_include(common)
 
 setup=chroot_setup
 unshare=0
+userspec=''
+chroot_args=()
```

Comment:
The commit mixes functional and cosmetic changes. It also introduces promotes local variables to global scope eg `chroot_args` - that should either be voided to documented.

I would suggest fixing the functional/shellcheck issues - one commit per script. Then doing the same on the cosmetic side.